tests/myst/exec-package/Makefile

```diff
@@ -5,6 +5,7 @@ APPDIR = appdir
 APPNAME = hello
 CFLAGS = -fPIC -g
 LDFLAGS = -Wl,-rpath=$(MUSL_LIB)
+LDFLAGS += -L$(LIBDIR) -lopenenclave -L$(BUILDDIR)/mbedtls/lib -lmbedcrypto
```

Comment:
Do you really need this?

Reply:
Modified to only include -lopenenclave - needed by
extern const void* __oe_get_enclave_start_address(void);
extern const void* __oe_get_enclave_base_address(void);